sambacc/samba_cmds.py

```diff
@@ -36,6 +36,14 @@ def get_samba_specifics() -> list:
     return out
 
 
+def smbd_stdout_opt() -> str:
+    opt = "--log-stdout"
+    opt_lst = get_samba_specifics()
+    if "smbd_cli_debug_output" in opt_lst:
```

Comment:
Let's make all the entries we can look for in `SAMBA_SPECIFICS` top level variables so that its easy for future readers of the code to find all the possible values that we look for.

Reply:
Yes. Good thought..